poetichardware-spring2020.md

```diff
@@ -0,0 +1,132 @@
+- Intro to circuit. Power, ground, control, resistance.
+- Electronic control. The transistor + logic.
+- Calculator/clock/time based circuits/seven segment disaplays.
+- Building an Arduino from scratch.
+- Analog input (sensors).
+- Motors + mechanisms.
+
+### Meet the teachers
+
+![](/static/img/poetichardware/cwandt.jpg) 
+
+[CW&T](https://cwandt.com/) is the art and design practice of Che-Wei Wang & Taylor Levy. 
+
+Che-Wei Wang [pron. sey-wey] is an artist, designer & architect with expertise in computational and generative design, fabrication technologies, electronics, CNC machining, and metal manufacturing. The results range from architecture & sculpture to interactive installations & mobile apps. He is the winner of the 2003 SOM fellowship and the Young Alumni Achievement Award from Pratt Institute. Che-Wei has taught courses on design, time, creative computing, and inflatables, at various institutions. He is an alumnus of MIT Media Lab, ITP at NYU, and Pratt Institute.
+
+Taylor Levy [pron. tey-ler] is an artist & designer with a penchant for taking things apart, understanding how they work, and then putting them back together in a way that exposes their inner workings.The results take on a variety of forms from low-tech electronic sculpture to high-tech software & other executions. She has work on view at The Leonardo Museum of Science and Technology and was a resident at Fabrica Interactive in Treviso, Italy. She is an alumna of MIT Media Lab, ITP at NYU, and Vassar College.
+
+![](/static/img/poetichardware/symbiosis.gif) 
+
+[Sebastián Morales](https://www.adorevolution.com/) is an artist, engineer and researcher based in NYC. He develops interactive works at the intersections of robotics, digital culture and living systems. He is currently a new media artist in residence at Mana Contemporary in NJ working on speculative symbiotic relationships between single cells and internet bots. Creating estuaries for them to evolve into new hybrid forms.
+
+![](/static/img/poetichardware/tiri.jpg) 
+
+[Tiri Kananuruk](http://xxx.tiri.xxx/) is a performance and new media artist. She holds a BA in exhibition design from Chulalongkorn University in Bangkok, and an MPS from the Interactive Telecommunications Program at New York University. Her work explores the manipulation of sound in the context of technological consumerism, examining human relationships through the use of transmitted signals, machine learning, natural language processing and bodily movement.  
+
+This class is organized by Taeyoon Choi of the School for Poetic Computation. 
+
+### ​​I’m still not sure if I should take this class?!!?
+​​
+​​If you already have a clear electronics project in mind, like building a drone or a complex I.O.T. system, this may not be the right class for you. However, if you want to learn about electronics + hardware through the lens of poetry, self-expression and quirky thoughtfulness, we look forward to working a playing together.
```

Comment:
```suggestion
​​If you already have a clear electronics project in mind, like building a drone or a complex I.O.T. system, this may not be the right class for you. However, if you want to learn about electronics + hardware through the lens of poetry, self-expression and quirky thoughtfulness, we look forward to working and playing together.
```